app/views/admin/instances/_instance.html.haml

```diff
@@ -7,6 +7,10 @@
       %small
         - if instance.domain_block
           = instance.domain_block.policies.map { |policy| t(policy, scope: 'admin.instances.content_policies.policies') }.join(' · ')
+          - if instance.domain_block.public_comment.present?
+            %span.comment.public-comment "#{t('admin.domain_blocks.public_comment')}: #{instance.domain_block.public_comment}"
+          - if instance.domain_block.private_comment.present?
+            %span.comment.private-comment "#{t('admin.domain_blocks.private_comment')}: #{instance.domain_block.private_comment}"
```

Comment:
There are extra quotes around the lines, this should be fixed with the following:
```suggestion
          - if instance.domain_block.public_comment.present?
            %span.comment.public-comment #{t('admin.domain_blocks.public_comment')}: #{instance.domain_block.public_comment}
          - if instance.domain_block.private_comment.present?
            %span.comment.private-comment #{t('admin.domain_blocks.private_comment')}: #{instance.domain_block.private_comment}
```

Reply:
Thank you!